solutions/core-landing-zone/setters.yaml

```diff
@@ -108,6 +108,8 @@ data:
   # Security Logs Bucket
   # customization: required
   security-log-bucket: security-log-bucket-12345
+  # Security incident log bucket
```

Comment:
might be worth adding that this is a storage bucket, and that the name must be globally unique across all GCP

Reply:
Please verify and resolve if good.